_includes/softwareTable.html

```diff
@@ -1,165 +1,176 @@
 <div class="row">
 
-<!-- Filters -->
```

Comment:
Is this all just a tab change or does this change the code in any way?

Reply:
This is just a mysterious tab change